src/chapters/control-flow-and-collections/dictionary.rst

```diff
@@ -0,0 +1,250 @@
+.. index:: ! Dictionary
```

Comment:
there is still a lot of usage of "map" on this page - i think a reread could be useful

Reply:
Done!